after transitioned event button click is removed

arvind248 opened the pull request:
I have  added code to remove click event of button instead of removing transitioned event to stop the user to interact again.

CLAassistant commented:
[![CLA assistant check](https://cla-assistant.io/pull/badge/not_signed)](https://cla-assistant.io/javascript-tutorial/en.javascript.info?pullRequest=2708) <br/>Thank you for your submission! We really appreciate it. Like many open source projects, we ask that you sign our [Contributor License Agreement](https://cla-assistant.io/javascript-tutorial/en.javascript.info?pullRequest=2708) before we can accept your contribution.<br/><sub>You have signed the CLA already but the status is still pending? Let us [recheck](https://cla-assistant.io/check/javascript-tutorial/en.javascript.info?pullRequest=2708) it.</sub>

iliakan commented:
Please elaborate, what exactly is wrong?

arvind248 commented:
actually, when button is clicked again , the circle is rendered again on the earlier circle it seems to be nothing happened on click. 
if the intention is to stop user clicking more then one, I have added ``` btn.removeEventListener("click",go);   ```  to remove click event of button and removed this line ```div.removeEventListener('transitioned',handler)  ```
